Sources/StreamChatUI/Utils/AudioPlayer/AudioPlayingDelegate.swift

```diff
@@ -0,0 +1,15 @@
+//
+// Copyright © 2023 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+
+/// An object that the audioPlayer can communicate updates regarding its context
+public protocol AudioPlayingDelegate: AnyObject {
+    /// The audioPlayer will calls this one to inform its delegate that the the playback's context has
+    /// been updated.
```

Comment:
Some typos:
```suggestion
    /// The audioPlayer will call this one to inform its delegate that the playback's context has
    /// been updated.
```

Tip: You can turn on the spell checker in Xcode. It will help a lot 👌